content/widgets/current-refinements.md

```diff
@@ -52,8 +52,12 @@ alt1: with empty excludedAttributes
 classes:
   - name: .ais-CurrentRefinements
     description: the root div of the widget
+  - name: .ais-CurrentRefinements--noRefinement
+    description: the root div of the widget with no refinement
   - name: .ais-CurrentRefinements-list
     description: the list of all refined items
+  - name: .ais-CurrentRefinements-list--noRefinement
+    description: the list with no refinement
```

Comment:
Why do we have it on the list too?

Reply:
This is based on react-instantsearch's `<CurrentRefinements>`, and is implemented in react-instantsearch-hooks-web as well.

I don't find this one useful, but we have to take into account its existence and the fact that it might be used by people. One thing we can do if we don't want it, is to remove it from the hooks-web component (+ doc) and it will ultimately replace the other component in the next major. Wdyt?

ccing @Haroenv since he was in the original discussion

Reply:
I guess it doesn't bring much indeed, feel free to remove